docs/examples/custom/README.ipynb

```diff
@@ -335,6 +335,26 @@
     "With that goal in mind, the rough outline of steps will be to first build a custom image containing our code, and then deploy it.\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Specifying requirements\n",
+    "MLServer will automatically find your requirements.txt file and install necessary python packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile models.py\n",
```

Comment:
Should this line be `%%writefile requirements.txt` instead?

Reply:
Sure, That's why I shouldn't do all the pull requests late at night. :D
Should work now. Please check.